src/pfe/portal/modules/LoadRunner.js

```diff
@@ -178,7 +178,13 @@ module.exports = class LoadRunner {
           metricsJson.desc = this.runDescription;
         }
         try {
-          await fs.writeJson(this.workingDir + '/metrics.json', metricsJson, { spaces: '  ' });
+          // eslint-disable-next-line no-sync
```

Comment:
So long as we've got the awaits correct there's no need to move to the Sync versions of these methods. It looks like you fixed not awaiting for the whole function below. There's also no need to check if the directory exists, just do:
```
await fs.ensureDir(this.workingDir);
await fs.writeJson(this.workingDir + '/metrics.json', metricsJson, { spaces: '  ' });
```